redisless/src/server.rs

```diff
@@ -184,6 +184,28 @@ fn run_command_and_get_response<T: Storage>(
             Command::Del(k) => {
                 let total_del = unlock(storage).del(k.as_slice());
                 format!(":{}\r\n", total_del).as_bytes().to_vec()
+            },
+            Command::Incr(k) => {
+                match unlock(storage).get(k.as_slice()) {
+                    Some(value) => {
+                        if let Ok(mut int_val) = std::str::from_utf8(value).unwrap().parse::<i64>() {
+                            int_val += 1;
+                            let new_value = int_val.to_string().into_bytes();
+                            unlock(storage).set(k.as_slice(), new_value.as_slice());
+                            let res = format!(":{}\r\n", int_val);
+                            res.as_bytes().to_vec()
+                        } else {
+                            let res = format!("-WRONGTYPE Operation against a key holding the wrong kind of value}}");
+                            res.as_bytes().to_vec()
```

Comment:
you can to `b"-WRONGTYPE Operation against a key holding the wrong kind of value".to_vec()` directly